snowfakery/utils/collections.py

```diff
@@ -67,3 +67,11 @@ def copy(self):
 
     def __repr__(self):
         return str(dict(self.items()))
+
+
+class SortedSet(OrderedDict):
```

Comment:
How about calling it OrderedSet? SortedSet makes it sound like it will do the equivalent of sort() as its contents are mutated.

Could it simply extend `dict` now that we dropped support for Python versions where dict is not ordered?